tests/cache/models/subjects/subjectCRUD.js

```diff
@@ -281,24 +282,27 @@ describe('tests/cache/models/subjects/subjectCRUD.js >', () => {
     .catch(done);
   });
 
+  it('removeFromRedis removes all the related samples ' +
```

Comment:
not sure if this file is the best place to test subjectUtils, but subjectUtils need to be tested. 
perhaps we can move the new test to a different file, once we get enough tests on subjectUtils.

Reply:
one test is enough for a new file :)